bokeh/server/tornado.py

```diff
@@ -69,6 +73,7 @@ def __init__(self, applications,
                     route = p[0]
                 else:
                     route = key + p[0]
+                route = self._prefix + route
```

Comment:
I'm open to suggestion, I was not sure which choice would be more appropriate. I don't really like the path we server BokehJS under to be perfectly honest.